app/services/concerns/course/user_invitation_service/email_invitation_concern.rb

```diff
@@ -161,7 +161,8 @@ def add_existing_users(users)
       if course_user
         existing_course_users << course_user
       else
-        new_course_users << @current_course.course_users.build(user: user[:user], name: user[:name],
+        role = user[:role] || Course::UserInvitation.roles[:student]
+        new_course_users << @current_course.course_users.build(user: user[:user], name: user[:name], role: role,
```

Comment:
If we set that the `:student` role be the default in the `course_user` model, then actually the above line `role = user[:role] || Course::UserInvitation.roles[:student]` is not necessary, because the model will handle `nil` values of role.